AppInspector.Common/Utils.cs

```diff
@@ -14,10 +14,14 @@ public static class Utils
     /// </summary>
     /// <param name="modifiers"></param>
     /// <returns>A RegexOptions object with the correct modifiers set</returns>
-    public static RegexOptions RegexModifierToRegexOptions(IList<string> modifiers)
+    public static RegexOptions RegexModifierToRegexOptions(IList<string> modifiers, bool enableNonbacktrackingRegex = false)
     {
         RegexOptions opts = new();
         opts |= RegexOptions.Compiled;
+#if NET7_0_OR_GREATER
+        if (enableNonbacktrackingRegex)
+            opts |= RegexOptions.NonBacktracking;
+#endif
         foreach (var modifier in modifiers)
```

Comment:
I thinl this switch list is missing two cases now that should be handled. `b` in the case that non backtrack is set and `nb` in either case. The `b` can be set with the `&= ~` technique and the nb with `|=`. Specifically for cases where it's set individually in the rule rather than by parameter.

Reply:
Changed to `&= ~`